README.md

```diff
@@ -1,7 +1,8 @@
 # Parasoft Findings for Azure DevOps
 
-The Parasoft Findings extension is designed to be used in a post test-execution build step that integrates static analysis, functional and unit test reports generated by Parasoft tools into your Azure DevOps project. The task will collect static analysis and test results from the Parasoft report XML files.
+The Parasoft Findings extension is designed to be used in a post test-execution build step that integrates static analysis, functional and unit test reports generated by Parasoft tools into your Azure DevOps project. The task will collect static analysis, test results and code coverage from the Parasoft report XML files.
 - For test results, the number of passed/failed tests and result details will be displayed. Trending data in a graph for a simple visualization is also available.
 - For static analysis results, this plugin has been integrated with the [SARIF SAST Scans Tab](https://marketplace.visualstudio.com/items?itemName=sariftools.scans) to visualize static analysis results. Navigation to rule documentation is also available.
+- For code coverage, this plugin will generate cobertura report with line coverage information from Parasoft report XML, and display in Azure DevOps.
```

Comment:
I think we may not need to mention the intermediate format (cobertura) here. User only needs to know line coverage will be displayed and maybe the details is under "Code Coverage" tab.